incubator/backup/README.md

```diff
@@ -2,31 +2,15 @@
 
 This chart starts a NuoDB database backup on existing NuoDB storage managers in a Kubernetes cluster using the Helm package manager.
 
-## TL;DR;
+## Command
 
 ```bash
-helm install nuodb/backup
+helm install nuodb/backup --name backup [--set parameter] [--values myvalues.yaml]
 ```
 
-## Prerequisites
-
-- Kubernetes 1.9+
-- An existing NuoDB Admin cluster has been provisioned
-
```

Comment:
these prerequisites are still true and should be kept here.

Reply:
at the bare minimum this page needs to link to another page that mentions the minimums

Reply:
These have now been covered elsewhere. Pre-reqs have been moved to a single location on the stable page rather than having copies on approximately 10 stable and incubator sub pages.

Reply:
I have explained this in person, but let me write this here again. These documents are not hierarchical. Any of these small READMEs can be landing pages for a new user. They need to be reasonable stand-alone documents that provide the reader with enough information about the chart or a link to a more complete set of docs.

We should
1) keep this section
2) change K8s to 1.14
3) link to the github readme page

Reply:
Would they ever get to a low-level chart page and not have the stable page? I have put the pre-reqs once on this page rather than all sub folder pages.

Reply:
Yes, that is what I am saying :)

Reply:
Resolved. I have added to each sub chart a prerequisite link back to the main software requirements section on the Helm Chart main page.